README.md

```diff
@@ -6,6 +6,7 @@ A set of streaming data parsers based on [fs2][fs2].
 Following modules are available:
   - `fs2-data-json`: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-json_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-json_2.13) A JSON parser and manipulation library
   - `fs2-data-json-circe`: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-json-circe_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-json-circe_2.13) [circe][circe] support for parsed JSON.
+  - `fs2-data-json-contextual`: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-json-contextual_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-json-contextual_2.13) [contextual][contextual] support for statically checked JSON interpolators.
```

Comment:
I would suggest to use a name which indicates what the user can do with the module rather than what dependencies it has, that's implementation detail to me. So maybe `-interpolators` or maybe the almighty `-utils`?

Reply:
Why not.